apps/api/v2/src/modules/users/users.repository.ts

```diff
@@ -181,6 +181,14 @@ export class UsersRepository {
 
     return user?.defaultScheduleId;
   }
+
+  async getOrganizationUsers(organizationId: number) {
+    return this.dbRead.prisma.user.findMany({
+      where: {
+        organizationId,
```

Comment:
organizationId is deprecated, we should look at the org in the profile

Reply:
or directly call 

this.dbRead.prisma.profile.findMany({
      where: {
        organizationId
        }
        })

Reply:
> organizationId is deprecated, we should look at the org in the profile

@ThyMinimalDev we should probably add this note in the schema as well, I don't think most of us are aware of it.

Reply:
indeed

Reply:
Done in [refactor: rely on profiles instead of user.organizationId](https://github.com/calcom/cal.com/pull/15584/commits/b2e7397916b9851f05ff2c4b8a290b233e39def3). Added also profile to tests (without having profile tests fails so code works as expected) and also added comment in prisma schema.